Cython/Utility/ImportExport.c

```diff
@@ -3,16 +3,31 @@
 static PyObject *__Pyx_ImportDottedModule(PyObject *name, PyObject *parts_tuple); /*proto*/
 static PyObject *__Pyx_ImportDottedModule_WalkParts(PyObject *module, PyObject *name, PyObject *parts_tuple); /*proto*/
 
+
 /////////////// ImportDottedModule ///////////////
 //@requires: Import
 
+#ifndef _ImportDecRef
+#if CYTHON_COMPILING_IN_LIMITED_API
+#define _ImportDecRef(obj) Py_DecRef(obj);
```

Comment:
I'd just use the macros in ModuleSetupCode.c - they're available absolutely everywhere so you don't need specific macros here.